lib/streamlit/elements/streamlit_plotly_theme.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
I think the placement of this file directly in the `elements` folder is not optimal. I don't think we have a very good place for this yet, but I would prefer to move it into the `elements/lib` folder. I'm not 100% sure about the intention of the `lib` folder, but it feels like a better place. IMO the elements folder itself should be reserved to actual elements.